_data/used_software.yml

```diff
@@ -38,11 +38,10 @@
   website: http://home.imf.au.dk/jensen/software/gfan/gfan.html
 
 - name: Factory
-  website: ftp://www.mathematik.uni-kl.de/pub/Math/Singular/Factory
+  website: https://www.mathematik.uni-kl.de/ftp/pub/Math/Singular/Factory
```

Comment:
@hannes14 is the link OK?

In any case, the link on https://www.singular.uni-kl.de/Manual/latest/sing_1.htm is also outdated (it's the same as the old one above)

Reply:
I guess it could also link to https://www.singular.uni-kl.de/dox/html/factory_page.html

Reply:
@hannes14 while we are at it, I made https://github.com/Singular/Sources/pull/993 (which does *not* attempt to resolve this URL, but a bunch of others on the same manual page)